scout/commands/update/disease.py

```diff
@@ -16,7 +16,7 @@
 from flask.cli import current_app, with_appcontext
 
 from scout.constants import UPDATE_DISEASES_RESOURCES
-from scout.load.hpo import load_disease_terms
+from scout.load.phenotype import load_disease_terms
```

Comment:
I think the old scout.load.hpo should be named something general, because at the moment contains a function to load both HPO and diagnoses, one to load the HPO and one to load the OMIM (diagnoses), That's why I named it phenotype